Assets/Scripts/UI/CharacterButtonHandler.cs

```diff
@@ -13,7 +14,13 @@ public class CharacterButtonHandler : MonoBehaviour
     [Header("Hover")]
     public GameObject hover;
 
+    [Header("Lock Image")]
+    public GameObject lockImage;
+    public int characterNum;
```

Comment:
아예 inspector에서 Character를 선택할 수 있도록,
```suggestion
    public Character character;
```
로 두는건 어때요?